tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -188,6 +197,24 @@ Verify Favorite Resource Cards
     Favorite Items Should Be Listed First When Sorted By    ${favorite_ids}    duration
     [Teardown]    Remove Items From favorites    @{favorite_ids}
 
+Verify Notifications Are Shown When Notebook Builds Have Not Started
+    [Documentation]     Verifies that Notifications are shown in RHODS Dashboard when Notebook builds haven't started
+    [Tags]    Sanity
+    ...       ODS-1347
+    ...       Tier1
```

Comment:
Sorry I didn't think about this yesterday but as  execution is 48 minutes I think it should be in Tier2, not Tier1

Reply:
I would say Tier3 where all the long running tests from all the other Tiers are placed

Reply:
updated.